subiquity/server/controllers/filesystem.py

```diff
@@ -998,16 +998,10 @@ async def finish_install(self, context):
         )
 
     async def guided_POST(self, data: GuidedChoiceV2) -> StorageResponse:
-        log.debug(data)
-        await self.guided(data)
-        if not data.capability.supports_manual_customization():
-            await self.configured()
-        return self._done_response()
+        raise NotImplementedError
```

Comment:
Better to just remove these from the API than do this?

Reply:
Yes, no problem!